src/schema.js

```diff
@@ -205,7 +211,12 @@ export default class DataStructureGenerator {
 
     let element;
 
-    if (schema.enum) {
+    if (schema.$ref) {
+      // element = new RefElement(idForDataStructure(schema.$ref));
```

Comment:
IMO this should be a reference element. However doing so doesn't work with attributes kit 😞.